indico/modules/designer/models/templates.py

```diff
@@ -95,6 +100,12 @@ class DesignerTemplate(db.Model):
         foreign_keys=background_image_id,
         post_update=True
     )
+    backside_template = db.relationship(
+        'DesignerTemplate',
+        lazy=True,
+        foreign_keys=backside_template_id,
```

Comment:
This may not be necessary since you already set `remote_side` and there is only one FK linking the table to itself